generateSettings.ts

```diff
@@ -215,4 +232,26 @@ text += ';\n';
 text += `export const settingsSchema = ${JSON.stringify(createSchema(true))};`;
 writeFileSync(`lib/settingsGenerated.ts`, text);
 
-writeFileSync(`settings.schema.json`, JSON.stringify(createSchema(false)));
\ No newline at end of file
+writeFileSync(`settings.schema.json`, JSON.stringify(createSchema(false)));
+
+let readmeText = readFileSync('README-input.md', {encoding: 'utf-8'}) + '\n';
+function ISchemaObject(schema: Schema): schema is ISchemaObject {
+  return schema.type === 'object' && Object.values(schema.properties).length > 0;
```

Comment:
This does not properly differentiate against `ISchemaObjectLiteral`. It just assumes that all objectLiterals have an empty `properties` which does not need to be the case I believe